ush/oceanice_nc2grib2.sh

```diff
@@ -45,23 +45,23 @@ function _ice_nc2grib2 {
 # tables in wgrib2 v2.0.8:
 
 #  -import_netcdf "${infile}" "hs_h" "0:1:${latlon_dims}" \
-#    -set_var ??? -set center 7 \
+#    -set_var SNVOLSI -set center 7 \
```

Comment:
Just a note here that this (and the others below) are commented out.

Reply:
Yes, it is intentionally to put on hold. This is a todo item for future update.